php/class-wp-cli.php

```diff
@@ -590,7 +590,16 @@ public static function add_command( $name, $callable, $args = array() ) {
 			self::get_runner()->register_early_invoke( $args['when'], $leaf_command );
 		}
 
-		\WP_CLI::debug( "Adding command: {$name}", 'commands' );
+		if ( ! empty( $parent ) ) {
+
```

Comment:
```suggestion
```

Reply:
I'd remove the 4 blank lines here, the code complexity or importance does not warrant so much real estate.